subcommittees/proposal_instructions.md

```diff
@@ -0,0 +1,35 @@
+##Software Carpentry Subcommittee & Task Force Program
+
+One of the greatest things about Software Carpentry is the huge diversity of people and ideas that have come together to make this project what it is: a worldwide collaboration of folks working together to build software skills. Your contributions to the community have made Software Carpentry a huge success - but we can go even further. To meet the needs of the growing global community, the 2016 Steering Committee is creating the opportunity for you to make Software Carpentry what your community needs it to be, via our **newly revamped Subcommittee & Task Force Program**.
```

Comment:
Thanks for all your comments, @wking! The logic for this structure is to provide a simple document up front for folks to land on that mostly just emphasizes the importance of getting in touch with ideas (that's the `README`) - and leaves more detailed criteria to the fine print. Also, you're right that there's a bit of repeated preamble between both documents, but this is to provide context regardless of where folks are coming from (ie people may in fact encounter `proposal_instructions.md` directly in future).